app/views/govuk_publishing_components/components/docs/fieldset.yml

```diff
@@ -30,5 +30,14 @@ examples:
         <input type="radio" id="html-legend-yes" name="html-legend">
         <label for="html-legend-yes">Yes</label>
 
+        <input type="radio" id="html-legend-no" name="html-legend">
+        <label for="html-legend-no">No</label>
+  without_legend:
+    data:
+      text: |
+        <!-- Use the radio component, this is hardcoded only for this example -->
+        <input type="radio" id="html-legend-yes" name="html-legend">
```

Comment:
The ids used here and below - html-legend-yes and html-legend-no - are already used in another example in the same file, so the component guide is reporting this as an error, when it isn't really. Best to use unique ids in data for each example.